manifests/config.pp

```diff
@@ -75,6 +75,7 @@
   $backup_cron_enable = $::gitlab::backup_cron_enable
   $backup_cron_minute = $::gitlab::backup_cron_minute
   $backup_cron_hour = $::gitlab::backup_cron_hour
+  $roles = $::gitlab::roles
```

Comment:
is there a way to make this var name more descriptive? perhaps `$ha_roles` or something like that?

Reply:
For the most part, this module uses variables that are consistent with the names of the config keys expected by the product. This allows users to reference the product documentation for how a config is expected to change the behavior of the product, and removes ambiguity/uncertainty by the implementor on whether or not _our_ name for a config is really _some other_ config for the [gitlab.rb.template](https://gitlab.com/gitlab-org/omnibus-gitlab/blob/master/files/gitlab-config-template/gitlab.rb.template), which is what this module configures.